packages/client/hmi-client/src/components/home/tera-project-menu.vue

```diff
@@ -1,5 +1,6 @@
 <template>
-	<Button icon="pi pi-ellipsis-v" rounded text @click.stop="toggle" :disabled="isEmpty(projectMenuItems)" />
+	<span v-if="isForking">Forking...</span>
+	<Button v-else icon="pi pi-ellipsis-v" rounded text @click.stop="toggle" :disabled="isEmpty(projectMenuItems)" />
```

Comment:
What is the requirement? Do we need all other users to also know if the project is being forked?
e.g. do we need an actual flag in the data structure or is it okay that this is a browser session only thing?

Reply:
This is like a loading/spinner thing. @kbirk told me that there is a few second wait before the forked project opens.